packages/layout/src/tree-layout/data-adapters/data-parser.js

```diff
@@ -0,0 +1,21 @@
+import { DataPoint } from './data-point';
+
+export class DataParser {
+    constructor (data) {
+        this.data = data;
```

Comment:
For class variables, use a class method to get and set this variable.
For example, 

use this._data = data
and have a class method like data(){...acts as getter and setter}